Snakefile

```diff
@@ -341,6 +341,29 @@ if config["enable"].get("retrieve_cost_data", True):
             move(input[0], output[0])
 
 
+rule build_demand:
+    input:
+        elec_network="networks/" + RDIR + "elec.nc",
+        regions="resources/" + RDIR + "bus_regions/regions_onshore.geojson",
+        load=load_data_paths,
+        #gadm_shapes="resources/" + RDIR + "shapes/MAR2.geojson", 
+        #using this line instead of the following will test updated gadm shapes for MA.
+        #To use: downlaod file from the google drive and place it in resources/" + RDIR + "shapes/
+        #Link: https://drive.google.com/drive/u/1/folders/1dkW1wKBWvSY4i-XEuQFFBj242p0VdUlM
+        gadm_shapes="resources/" + RDIR + "shapes/gadm_shapes.geojson",
+    output:
+        "networks/" + RDIR + "elec_1.nc",
```

Comment:
Output would be ideally a `csv file`

So in a way, what goes in is the GEGIS hourly time-series for a region e.g. Africa (which has hourly data for each country). What is the expected output is then a `csv` with the `load` distributed across the regions within a country. Debug the variable `load` from the below function which should be contained in the csv.

`n.madd("Load", substation_lv_i, bus=substation_lv_i, p_set=load)`

This function should remain in `add_electricity`

Reply:
Done, the csv has the snapshots as rows and buses as columns.